.github/workflows/ci.yml

```diff
@@ -81,12 +81,12 @@ jobs:
                 cargo test --release -p ac-testing-sync,
 
                 # Clippy
-                cargo clippy -p substrate-api-client -- -D warnings,
-                cargo clippy -p substrate-api-client --no-default-features -- -D warnings,
-                cargo clippy -p substrate-api-client --all-features --examples -- -D warnings,
+                cargo clippy --workspace --exclude ac-keystore --exclude test-no-std -- -D warnings,
```

Comment:
Is the exclude ac-keystore still necessary?

Reply:
Yeah, you are right. I removed it